index.html

```diff
@@ -44,8 +48,167 @@ <h1 class="prmrytxt1">
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+    <div class="port1">
+      <img src="images/card3pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Tonic</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p>
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>    
+    <div class="port1">
+      <img src="images/card4pic.png" alt="Tonic">
+      <div>
+        <h2 class="title">Multi-Post <br> Stories</h2>
+        <div>
+          <span class="clnt">CANOPY</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">Back End Dev</span>
+          <img src="images/circleicn.png" alt="Circle" class="crcl">
+          <span class="deets">2015</span>
+        </div>
+      </div>
+      <p >
+        A daily selection of privately personalized reads; no accounts or sign-ups required.
+      </p>
+      <ul class="tags">
+        <li class="tgs">html</li>
+        <li class="tgs">css</li>
+        <li class="tgs">javascript</li>
+      </ul>
+      <div>
+        <button class="prjctbtn">
+          <span class="btntxt">See Project</span>
+        </button>
+      </div>
+    </div>
+  </section>
+
+  <!---About Section-->
+  <section class="about">
+    <i class="prymrytxt1">
+      About <br> Myself
+    </i>
+    <p>
+      Hello I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+    <a href="#"><h2 class="letsconnect">LET'S CONNECT</h2></a>
+    <ul class="socials">
+      <li><a href="#" target="_blank"><img src="images/github.png" alt="github"></a></li>
+      <li><a href="#" target="_blank"><img src="images/linkedin.png" alt="linkedin"></a></li>
+      <li><a href="#" target="_blank"><img src="images/twitter.png" alt="twitter"></a></li>
+      <li><a href="#" target="_blank"><img src="images/cptM.png" alt="m"></a></li>
+      <li><a href="#" target="_blank"><img src="images/hand.png" alt="chill"></a></li>
+    </ul>
```

Comment:
[OPTIONAL] This is optional but while you're at it, why not include the correct links 🔗  of your socials? 😉